package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sagui",
-  "version": "5.0.0-rc.6",
+  "version": "5.0.0-rc.7",
```

Comment:
This was already in master, but I haven't pushed yet. Sorry. It is not actually part of this PR.